<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-11T19:40:46 CET" name="Default Suite" finished-at="2022-01-11T19:40:52 CET" duration-ms="5293">
    <groups>
    </groups>
    <test started-at="2022-01-11T19:40:46 CET" name="BuySharesTestFramework" finished-at="2022-01-11T19:40:52 CET" duration-ms="5293">
      <class name="RefreshTest">
        <test-method is-config="true" signature="setupSuite()[pri:0, instance:RefreshTest@56620197]" started-at="2022-01-11T19:40:46 CET" name="setupSuite" description="Setting up Suite" finished-at="2022-01-11T19:40:46 CET" duration-ms="15" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupSuite -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:RefreshTest@56620197]" started-at="2022-01-11T19:40:46 CET" name="setupTest" description="Setting up Test" finished-at="2022-01-11T19:40:46 CET" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method is-config="true" signature="setupMethod(org.testng.ITestResult,org.testng.ITestContext)[pri:0, instance:RefreshTest@56620197]" started-at="2022-01-11T19:40:46 CET" name="setupMethod" finished-at="2022-01-11T19:40:51 CET" duration-ms="4956" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name={null} status=CREATED method=RefreshTest.verifyProvidersURLNotBroken()[pri:6, instance:RefreshTest@56620197] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@41b0ae4c]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupMethod -->
        <test-method signature="verifyProvidersURLNotBroken()[pri:6, instance:RefreshTest@56620197]" started-at="2022-01-11T19:40:51 CET" name="verifyProvidersURLNotBroken" description="TC_006 - Verify Provider Urls not broken" finished-at="2022-01-11T19:40:52 CET" duration-ms="166" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.info(String)" because the return value of "ExtentReporters.ExtentFactory.getTest()" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.info(String)" because the return value of "ExtentReporters.ExtentFactory.getTest()" is null
at RefreshTest.verifyProvidersURLNotBroken(RefreshTest.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyProvidersURLNotBroken -->
        <test-method is-config="true" signature="cleanupMethod()[pri:0, instance:RefreshTest@56620197]" started-at="2022-01-11T19:40:52 CET" name="cleanupMethod" description="Cleaning up after Test Method" finished-at="2022-01-11T19:40:52 CET" duration-ms="151" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanupMethod -->
      </class> <!-- RefreshTest -->
    </test> <!-- BuySharesTestFramework -->
  </suite> <!-- Default Suite -->
</testng-results>
